timelock-impl/src/main/java/com/palantir/atlasdb/timelock/TimeLockResource.java

```diff
@@ -41,7 +41,7 @@
 import com.palantir.timestamp.TimestampManagementService;
 import com.palantir.timestamp.TimestampService;
 
-@Path("/{namespace: [a-zA-Z0-9_-]+}")
+@Path("/{namespace: (?!tl)[a-zA-Z0-9_-]+}")
```

Comment:
this refuses to match against `/tl`? not sure how hard it would be to ensure that we never hit this path accidentally, since it's an integration test

I imagine we want to roll out undertow across more endpoints, what if all conjure endpoints started off with `~c`? Or some starting symbol that won't match against this similar to how there is `.internal` for the paxos stuff? Then we don't have to change this line.

Reply:
While we're here, just thought of something: if we decide we want to continue at this level, this should probably be `(?!tl$)` instead, in case someone decides to name their client beginning with `tl`. As written this doesn't match e.g. `tlb` or `tldr` which is probably not the intention. I realise we're going for very short paths in general for perf reasons.

Reply:
Conjure endpoints cannot start with special characters.

Reply:
(have added $)